modules/devices/steamdeck/fde.nix

```
@@ -0,0 +1,105 @@
+{ config, lib, pkgs, utils, ... }:
+
+let
+  cfg = config.jovian.devices.steamdeck;
+in
+{
+  options = {
+    jovian.devices.steamdeck = {
+      fde = {
+        deckbd = {
+          enable = lib.mkOption {
+            default = cfg.enable;
+            defaultText = lib.literalExpression "config.jovian.devices.steamdeck.enable";
```

Oh, good thing I'm testing on my system, I missed this.

This can't be enabled by default. It should be considered an opt-in option from the end-user. Especially since it changes the existing semantics of the gamepad during boot.
```suggestion
            default = false;
```

(And also because it will cause a sudden `assertion` failure when not already using systemd stage-1.)